Hue.xcodeproj/project.pbxproj

```diff
@@ -611,6 +618,8 @@
 				PRODUCT_NAME = Hue;
 				SDKROOT = macosx;
 				SKIP_INSTALL = YES;
+				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
```

Comment:
It's been updated by Xcode's Migrator. I haven't changed the value. If there is a disadvantage to this, I can remove it.